services/squash-api/values-squash-sandbox.yaml

```diff
@@ -0,0 +1,91 @@
+squash-api:
+  replicaCount: 1
+
+  image:
+    repository: lsstsqre/squash-restful-api
+    tag: latest
+    pullPolicy: IfNotPresent
+
+  cloudSQLImage:
+    repository: gcr.io/cloudsql-docker/gce-proxy
+    tag: "1.17"
+    pullPolicy: IfNotPresent
+
+  redisImage:
+    repository: redis
+    tag: "6.0"
+    pullPolicy: IfNotPresent
+
+  # SQuaSH Cloud SQL instance connection name
+  instanceConnectionName: "squash-db-sandbox"
+
+  # Credentials for the SQuaSH Cloud SQL service account
+  cloudSQLInstanceSecret: "cloudsql-instance-credentials"
+
+  # SQuaSH DB credentials
+  squashDBSecret: "squash-db-credentials"
+
+  # Default SQuaSH API credentials
+  squashSecret: "squash-credentials"
+
+  # If "True", job datetime is obtained from the job metadata instead of using
+  # current time. Use this option to restore existing jobs to SQuaSH.
+  squashETLMode: ""
+
+  # S3 Bucket to upload verification jobs
+  s3BucketName: "squash-sandbox"
+
+  # AWS credentials
+  awsSecret: "squash-aws-credentials"
+
+  # InfluxDB URL
+  influxUrl: "http://influxdb.influxdb:8086"
+  influxDb: "squash-sandbox"
+
+  # InfluxDB credentials
+  influxSecret: "influxdb-auth"
+
+  # Redis URL
+  redisUrl: "redis://squash-sandbox-squash-api-redis.squash-sandbox:6379"
+
+  podSecurityContext:
+    runAsNonRoot: true
+    runAsUser: 1000
+    runAsGroup: 1000
+
+  securityContext:
+    capabilities:
+      drop:
+      - ALL
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    runAsUser: 1000
+
+  service:
+    type: ClusterIP
+    port: 8080
+
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: "nginx"
+      nginx.ingress.kubernetes.io/rewrite-target: /
```

Comment:
Does this annotation do anything? I wonder if you could drop it for simplicity.

Reply:
I couldn't reach the Flask app without it, I think I need it because I have the path base route to `/squash-api` and the app expects `/` I had a similar issue with the [InfluxDB ingress configuration](https://github.com/lsst-sqre/lsp-deploy/pull/343/commits/ae8cbe67cbdf4436aefb2d336a126a6219eecfaf).